v3.0/htmlControlsForm.html

```diff
@@ -0,0 +1,71 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Html_Control_Form</title>
```

Comment:
@AndiAJ I think we should keep `externalLinks.html` as is and just create a new page `HTMLforms.html` `HTML` caps

Reply:
Sure, thanks !

Reply:
Done !